### DISCUSSION TITLE
Multi level pricing

### DISCUSSION THREAD

**Discussion by zycabathware** [Erratums/ims-support#74] [Ideas]
We need an option to set different pricing per item. Something like, Price A, Price B etc 
So that for each item we can set different prices and then we can assign Price A to a customer, so he will always get the Price a benefits

**Comment by wholemet** [comment]
We are also looking for something similar. **But we need it as per quantity**.
Say like, for `1-5` quantity, `INR 100`, for `6-13` qty, `INR 90`, above 13 INR 87 like this
